app/src/main/java/com/itsaky/androidide/MainActivity.java

```diff
@@ -44,6 +49,7 @@ public class MainActivity extends StudioActivity {
   @Override
   protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
+    showInstallJdkSdkRequired();
     openLastProject();
```

Comment:
This opens the project even if the tools aren't installed. It should be called in the `else` block of `showInstallJdkSdkRequired()` method.

Reply:
Okay